pkg/cmd/admin/policy/cani.go

```diff
@@ -209,13 +226,18 @@ func (o *canIOptions) listAllPermissions() error {
 		if err != nil {
 			return err
 		}
-		rulesReviewStatus = whatCanYouDo.Status
 
+		policyRules = whatCanYouDo.Status.Rules
+		rulesReviewResult = whatCanYouDo
+	}
+
+	if o.Printer != nil {
+		return o.Printer.PrintObj(rulesReviewResult, o.Out)
 	}
 
 	writer := tabwriter.NewWriter(o.Out, tabwriterMinWidth, tabwriterWidth, tabwriterPadding, tabwriterPadChar, tabwriterFlags)
```

Comment:
> could you elaborate more?

Sure, I was suggesting that we should add a new handler func ([kind of like was done for buildConfigs](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/printer.go#L341), for example) that knows how to print a list of policy rules while taking human-readable printer options into account. We could re-use some of the code that now exists in [DescribePolicyRule](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/describer.go#L1300).